skepticoin/networking/utils.py

```diff
@@ -1,29 +0,0 @@
-import json
```

Comment:
no moving around of code while doing other things please

Reply:
I had to move these 3 functions into `peers.py` to prevent an import-cycle.
This cycle came into existence when adding the type annotations.

This should be the only moved code in this PR.